tests/h/cli/commands/authclient_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import mock
+import pytest
+
+from h import models
+from h.cli.commands import authclient as authclient_cli
+
+
+class TestAddCommand(object):
+
+    def test_it_creates_an_authclient(self, cli, cliconfig, db_session):
+        authclient = self._add_authclient(cli, cliconfig, db_session)
+        assert authclient
```

Comment:
Right, so my reasoning was a bit vague.
The documentation clearly [states that `first()`](http://docs.sqlalchemy.org/en/rel_1_1/orm/query.html?highlight=first#sqlalchemy.orm.query.Query.first) that it returns the first result or `None`. So we should check that it's not `None` instead of a truthy value.